### PR TITLE
[tempo] Correctly sets the Tempo version for 2.1.1

### PR DESCRIPTION
**Pull request by joe-elliott**
The appversion was updated in the chart, but the actual images use these tags for some reason.

### CODE REVIEW

**Review by faustodavid — APPROVED** [review]
LGTM, thank you @joe-elliott .

I have created the issue https://github.com/grafana/helm-charts/issues/2401 to avoid this in the future.

**Review by faustodavid — CHANGES_REQUESTED** [review]
Can you also update the chart?

File: charts/tempo/values.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ annotations: {}
 
 tempo:
   repository: grafana/tempo
-  tag: 2.0.1
+  tag: 2.1.1
```
Comment (faustodavid) [inline charts/tempo/values.yaml:15]: We can set these 2 tags to null and modify the statefulset.yaml to also get the version from the chart AppVersion

```
image: {{ .Values.tempo.repository }}:{{ .Values.tempo.tag | default .Chart.AppVersion | quote }}
```
and
```
image: {{ .Values.tempoQuery.repository }}:{{ .Values.tempoQuery.tag | default .Chart.AppVersion | quote }}
```

Comment (joe-elliott) [reply charts/tempo/values.yaml]: nice. totally agree with this.

**Review by faustodavid — APPROVED** [review]

**Pull request merged**